.travis.yml

```diff
@@ -21,6 +21,6 @@ before_install:
   - yarn --version
 script:
   - yarn lint
-  - yarn test
+  - yarn test -- --ci --coverage
```

Comment:
Moved these flags out of the script definition so that `yarn test` works as expected locally.